src/containers/FormikForm/AddRevisionDateField.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { FormEvent } from 'react';
+import Button from '@ndla/button';
+import { Input, FieldRemoveButton } from '@ndla/forms';
+import { useTranslation } from 'react-i18next';
+import { FieldInputProps } from 'formik';
+import { spacing } from '@ndla/core';
+import styled from '@emotion/styled';
+import { Switch } from '@ndla/switch';
+import { ArticleFormType } from './articleFormHooks';
+import InlineDatePicker from './components/InlineDatePicker';
+import { formatDateForBackend } from '../../util/formatDate';
+
+type RevisionMetaFormType = ArticleFormType['revisionMeta'];
+
+interface Props {
+  formikField: FieldInputProps<RevisionMetaFormType>;
+  name: string;
+  placeholder?: string;
+  onChange: Function;
+  value: string[];
+  showError?: boolean;
+}
+
+const Wrapper = styled.div`
+  margin-bottom: ${spacing.small};
+  display: flex;
+  > div {
+    &:first-of-type {
+      + div {
+        padding-left: ${spacing.normal};
+      }
+    }
+  }
+  &:empty {
+    margin: 0;
+  }
+`;
+
+const InputWrapper = styled.div`
+  min-width: 65%;
```

Comment:
```suggestion
  flex: 1;
```

La heller inputfeltet vokse og oppta resten av tilgjengelig plass i containeren.

Da må `width: 100%` fjernes fra dato-input og `width: 15%` fjernes fra slette-knapp.

Reply:
Hmm, syns datofeltet ble litt bredere enn det trengte når jeg gjorde det sånn, men la gå :smile: